src/etcd/deploy/etcd-delete/delete-data.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+# to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+# The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+
+echo "Clean the etcd's data on the disk"
+# Note: etcd data is stored at EmptyDir volume, when pods is deleted, the dir will be empty.
```

Comment:
How to persist user data?

Reply:
you mean etcd store data at which folder ? Or user how to do snapshot and restore data?

For 1, it mounts to /var/etcd :
![image](https://user-images.githubusercontent.com/5576848/46989011-efb2b500-d12d-11e8-8f29-1e04baba0f6f.png)

For 2, i mentioned at doc Data Stored on Etcd part. If operator want to backup restore data could follow that